plugins/in_syslog/syslog.c

```diff
@@ -227,6 +227,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_syslog, parser_name),
      "Set the parser"
     },
+    {
+      FLB_CONFIG_MAP_SIZE, "buffer_rcv_size", (char *)NULL,
```

Comment:
please rename it to `receive_buffer_size` (and adjust all other name references.

Reply:
I thought that the naming buffer_rcv_size was more consistent with other options prefixed with buffer_ (buffer_chunk_size/buffer_max_size).
Nevertheless, I'll do the change.